CHANGES.md

```diff
@@ -15,7 +15,12 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+- Geoserver: protect web interface and ows routes behind magpie/twitcher
+ 
+  Updates Magpie version to 3.35.0 in order to take advantage of the updated Geoserver Service.
+
+  See https://github.com/bird-house/birdhouse-deploy/issues/333 for details.
```

Comment:
Should be a bit more verbose about the "potentially breaking" changes:

- `geoserverwms` Magpie service deprecated (could need manual permission updates to be transferred to `geoserver` if used by some external configuration)
- `geoserver` having new secured access by default instead of using https://github.com/bird-house/birdhouse-deploy/tree/master/birdhouse/optional-components/test-geoserver-secured-access 
  - breaks instances that assumed fully-open access to `/geoserver` endpoint and its OWS requests
  - makes `optional-components/test-geoserver-secured-access` deprecated and `geoserver-secured` endpoint and service deprecated as well
- warning about `all-public-access` that will now apply additional permissions for geoserver as well
  (somewhat of an edge-case, but someone that was using `geoserver-sercured` + `all-public-access` technically had a "more secured" geoserver that what these changes provide, see https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/optional-components/test-geoserver-secured-access/config/magpie/permissions.cfg.template).
  Since the new permission set allows "writable" or "destructive" operations on layers, this could be an issue.

Reply:
Ok I can add some of these as warnings.

> makes optional-components/test-geoserver-secured-access deprecated and geoserver-secured endpoint and service deprecated as well

If these are now deprecated, I could move them to the deprecated-components directory as part of this PR. What would you suggest @fmigneault

Reply:
Yes, that would be good. Thank you.